src/components/Popover/Popover.jsx

```diff
@@ -67,6 +73,12 @@ Popover.propTypes = {
     'left-start',
     'left-end',
   ]),
+  /**
+   * If set, the popover will become controlled, meaning it will be hidden by default and will need a trigger to open.
+   * This sets the ID of the internal helper element for the popover.
+   * Assign the same ID to `popovertarget` of a trigger to make it open and close.
+   */
+  popoverHelperId: PropTypes.string,
```

Comment:
I'm unsure if `popoverHelperId` is a good name. I find it too general.

I think I would prefer something like `popoverTargetId`, `popoverAnchorId`, `popoverPlacementHelperId` or such.

I don't feel super strong about it, so I will leave this to others to consider and decide.

Reply:
I think we decided to call it `popoverTargetId` so it matches `popoverTarget`.